textproc/recode/Portfile

```diff
@@ -24,16 +24,29 @@ long_description    This recode program has the purpose of converting files \
 
 github.tarball_from releases
 
-depends_test        port:py27-cython
+set python.version  37
+set python.branch   [string range ${python.version} 0 end-1].[string index ${python.version} end]
+set python.prefix   ${frameworks_dir}/Python.framework/Versions/${python.branch}
+set python.bin      ${python.prefix}/bin/python${python.branch}
 
-depends_lib         port:gettext \
-                    port:libiconv
+depends_build \
+    port:python${python.version}
```

Comment:
I suggest to simply keep this at 20 spaces.

Reply:
changed.
note: now the second line doesn't respect the coding style (header)